### PR TITLE
fix formatting for the CentOS installation instructions

### PR DESCRIPTION
**Pull request by aaronmarkham**
## Description ##
This PR fixes the broken bullets on the CentOS installation instructions.

## Preview
http://34.201.8.176/versions/centos_setup/install/centos_setup.html

@frankfliu

### CODE REVIEW

**Comment by aaronmarkham** [comment]
Currently the preview here on github looks good. The numbers increment, but the [html output](http://34.201.8.176/versions/centos_setup/install/centos_setup.html) just has 1, 1, 1, 1, ...
😡

**Comment by aaronmarkham** [comment]
Added "Step x" to each since the html publish step seemed intent on messing up the numbering no matter what... and I don't want to play those ❄️ 🦌 reindeer 🦌 ❄️ games right now.

Preview is updated.

**Comment by anirudhacharya** [comment]
@mxnet-label-bot [pr-awaiting-review]

**Comment by harshp8l** [comment]
@mxnet-label-bot add [pr-awaiting-review]

File: docs/install/centos_setup.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Installing MXNet on CentOS and other non-Ubuntu Linux systems
 
-1. Install build tools and git on `CentOS >= 7` and `Fedora >= 19`:
+Step 1. Install build tools and git on `CentOS >= 7` and `Fedora >= 19`:
```
Comment (kalyc) [inline docs/install/centos_setup.md]: minor - styling `Step #` will make it the doc more readable

**Comment by lupesko** [comment]
@aaronmarkham @kalyc - did you verify it works as expected on CentOS? haven't seen anyone calling this as done.

**Comment by aaronmarkham** [comment]
Nope. Just fixing formatting.

**Comment by kalyc** [comment]
@mxnet-label-bot update [pr-awaiting-merge]

**Pull request merged**